sort.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+function bubbleSort($sort_array)
+{
+    if (empty($sort_array)) {
+        return [];
+    }
+
+    $array_indexes = count($sort_array);
+
+    for ($i = 0; $i < $array_indexes; $i++) {
+        if (in_array("", $sort_array)) {
+            continue;
+        }
```

Comment:
Please explain what exactly this if does.

Reply:
Es wird nach jeder Schleifen Iteration überprüft, ob ein leeres String in $sort_array gibt. Wenn es true ist, dann ignoriert er den leeren String und bricht die Iteration und fängt von vorne an.

Reply:
Nein, nicht _nach_, sondern _zu Beginn_. In welchem Fall hast Du dort einen leeren String?